cogs/reward.py

```diff
@@ -1,25 +1,36 @@
 import discord
 import re
 from discord.ext import commands
-from bfunc import roleArray, calculateTreasure, timeConversion 
+from bfunc import roleArray, calculateTreasure, timeConversion, commandPrefix
 
 class Reward(commands.Cog):
 		def __init__ (self, bot):
 				self.bot = bot
 
 		@commands.cooldown(1, 5, type=commands.BucketType.member)
 		@commands.command()
-		async def reward(self,ctx, timeString, tier):
+
+		async def reward(self,ctx, timeString=None, tier=None):
+				rewardCommand = f"`{commandPrefix}reward [XhYm] [tier]`"
+
 				def convert_to_seconds(s):
 						return int(s[:-1]) * seconds_per_unit[s[-1]]
 
+				channel = ctx.channel
+				if timeString is None:
+						await channel.send(content=rewardCommand + " Time is required.")
```

Comment:
Thanks for error checking here. I think it's weird that rewardCommand is here though. The bot is going to send a message saying "$reward [XhYm] [tier]" Time is required" which I think is a little weird.

Reply:
I thought it would be good to remind the user of the correct syntax needed for the command without having to navigate back through the main help. I've seen users miss a parameter and then guess a couple more times before getting it right.

Reply:
If you use a git command incorrectly, for example, the response will show the possible flags and correct usage for that command

Reply:
Ahh okay, in that case can you add this to the end of the sentence instead? For ex. "Time is required. Please follow this template + rewardCommand"